coalib/bearlib/abstractions/Linter.py

```diff
@@ -711,6 +714,8 @@ class GlobalLinterMeta(type(LinterBase), type(GlobalBear)):
 
     class GlobalLinterBase(LinterBase, GlobalBear, metaclass=GlobalLinterMeta):
 
+        LinterParent.register(LinterBase)
```

Comment:
There would be a filename conflict because we already have a file called `Linter.py` and shouldn't the class name be the same as the filename ?